_source/_posts/2021-09-08-sonar-qube-dotnet.md

```diff
@@ -0,0 +1,167 @@
+
+The**Issues** section lets you go deeper into analysis of the source code. SonarQube reports on all possible problems: from minor issues to critical bugs. You can explore every issue in the code that you should take into consideration, along with an explanation of the issue and an estimated time for solving it.
+
+{% img blog/sonar-qube-dotnet/sonar-issues-list.png alt:"Screenshot of Listed Issues In Source Code" width:"800" %}{: .center-image }
+
+## Additional Configuration and SonarQube Customization 
+
+SonarQube will run on almost any stack. For this reason,you may notice some unwanted notifications. Pre-set rules may not work correctly in all frameworks, and may not know how to handle all file extensions. 
+
+Adding custom rules and configuration is a straightforward process, similar to excluding files or paths in GitHub. You can apply custom rules globally for your system or locally for a single project. To be precise, you can:
+
+- Completely ignore some files or directories
+- Exclude files/directories from **Issues** detection (via specific rules or all of them),  but analyze all other aspects.
+- Exclude files/directories from **Duplications** detection but analyze all other aspects
+- exclude files/directories from **Coverage** calculations but analyze all other aspects
+
+In order to make adjustments, go to
+**Administration** > **General Settings** > **Analysis Scope** (global settings) or
+**Project Settings** > **General Settings** > **Analysis Scope** (local configuration).
+
+You can learn more about [project administration](https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/) in the SonarQube documentation.
+
+##Code Analysis Matters
+
+We just explored how to set up SonarQube quickly and without too much effort. The solution we stepped through in this tutorial was based on .NET, however, you can easily apply it to over 20 other popular programming languages.
+
+As a developer, it's important to stay focused on code quality, especially with tools like SonarQube that make it easier than ever to analyze code and learn best practices for code security and resilience. In the workplace, it's also essential for developers to produce code that integrates gracefully with the work of others—and with SonarQube as a teammate, coding can become even more of a collaborative team sport.  😉
+
+If you enjoyed this post, you might like related ones on this blog:
+
+- [A Developer's Guide To Docker - A Gentle Introduction](/blog/2017/05/10/developers-guide-to-docker-part-1)
+- [Build a .NET App Quickly with Docker](/blog/2019/11/04/build-net-app-quickly-with-docker)
+
+Follow Okta Developers for more great content and updates from the team! You can find us on [Twitter](https://twitter.com/oktadev), [Facebook](https://www.facebook.com/oktadevelopers), subscribe to our [YouTube Channel](https://youtube.com/c/oktadev), or start the conversation below.
+
```

Comment:
I'd remove all the line breaks at the end here.

Reply:
Agreed! I'm not sure what is the best way to go about this. Is there a way to remove them using the "Insert suggestion" UI here?

Reply:
I removed them by editing the file in your branch. :)

Reply:
Aha! Thank you.